src/routes/index.tsx

```diff
@@ -77,12 +79,13 @@ const styles = StyleSheet.create({
 function TabBarPrimaryButton() {
   const navigation = useNavigation();
   const { colors } = useThemeColors();
+  const selectedBrandStyle = useSelector((state: RootState) => state.configuration.selectedBrandStyle);
```

Comment:
```suggestion
  const selectedBrandStyle = useSelector((state: RootState) => state.configuration.selectedBrandStyle || colors.brandStyleOrange);
```

Reply:
Updated it. 55b5dc9f0af99d1f85951d3704744a830f4324ee a22bf175aec8ba26e175b450add53bc58bd3e3dc